### PR TITLE
COMMERCE-5313 Add CPMediaImpl where you can only pass groupId for company image

### PR DESCRIPTION
**Pull request by alecsloan**

### CODE REVIEW

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by alecsloan** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/util/CPMediaImpl.java [hunk 1]
```diff
@@ -62,6 +62,20 @@ public CPMediaImpl(FileEntry fileEntry, ThemeDisplay themeDisplay)
...
+
+		_downloadUrl = defaultUrl;
+
+		_id = 0;
+		_url = defaultUrl;
+
+		_thumbnailUrl = defaultUrl;
+
+		_mimeType = null;
+		_title = null;
```
Comment (dsitu) [inline modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/util/CPMediaImpl.java:76]: @alecsloan I think we can remove the new lines and sort this alphabetically.

**Review by dsitu — CHANGES_REQUESTED** [review]

File: modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/util/CPMediaImpl.java (hunk 1 above)
Comment (alecsloan) [reply modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/util/CPMediaImpl.java:76]: Hey @dsitu I can do alphabetical but I have to leave spaces after referencing defaultUrl

Comment (dsitu) [reply modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/util/CPMediaImpl.java:76]: yep that looks good to me, thanks!

**Comment by alecsloan** [comment]
ci:test:sf

**Review by dsitu — APPROVED** [review]

**Review by marco-leo — APPROVED** [review]

**Comment by ivicac** [comment]
ci:forward

**Pull request closed without merging**